firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1560,7 +1571,9 @@ class BrowserViewController: UIViewController,
 
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
-
+        if isToolbarTranslucencyRefactorEnabled {
```

Comment:
Instead of existing early for everything we should just not update the blur views I think.